### PR TITLE
Fea 185 task 197 & Fea 186 task 191

### PR DESCRIPTION
**Pull request by SHulce**

### CODE REVIEW

File: rest-server/src/model/aog.hpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            aog_json["name"] = web::json::value::string(name);
+            aog_json["agency"] = web::json::value::string(agency);
+            return aog_json;
+        }
+
+        static std::optional<Aog> from_json(web::json::value input) {
+            if (input.has_field("name") && input["name"].is_string() &&
+                input.has_field("agency") && input["agency"].is_string()) {
+                if (input.has_field("id") && input["id"].is_integer()) {
+                    return Aog {input["id"].as_integer(),
```
Comment (kchoadley) [inline rest-server/src/model/aog.hpp]: Not needed, integers default value is 0. String default value is empty string

File: rest-server/src/rest_server.cpp [hunk 2]
```diff
@@ -26,9 +30,12 @@ std::string getEnvVar(std::string const& key) {
 }
 int main(int argc, const char * argv[]) {
     LoggerInterface& logger(Logger::Instance());
-    logger.LogLevel(LogLevel::INFO);
+    logger.LogLevel(LogLevel::DEBUG);
     logger.Log(LogLevel::INFO, "Starting cadg rest server");
+    AogDao::Instance().SetConnectionString(
```
Comment (kchoadley) [inline rest-server/src/rest_server.cpp]: No, bad Shawn.

File: rest-server/src/services/aog_dao.cpp [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            nanodbc::statement statement(connection);
+            prepare(statement, NANODBC_TEXT(
+                    "update cadg.originator set originator_name=?, agency=? where originator_id=?;"));
+            nanodbc::result results;
+            nanodbc::string const originator_name = NANODBC_TEXT(aog.name);
+            statement.bind(0, originator_name.c_str());
+            nanodbc::string const agency = NANODBC_TEXT(aog.agency);
+            statement.bind(1, agency.c_str());
+            const int *originator_id = &aog.id;
+            statement.bind(2, originator_id);
```
Comment (kchoadley) [inline rest-server/src/services/aog_dao.cpp]: Just use `&aog.id` here

File: rest-server/src/model/aog.hpp (hunk 1 above)
Comment (SHulce) [reply rest-server/src/model/aog.hpp]: in-line construction

**Pull request merged**